crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -266,9 +267,18 @@ def run_installer
 
       def restore_database
         Rails.logger.debug "Restoring Crowbar database"
-        SerializationHelper::Base.new(YamlDb::Helper).load(
-          @data.join("crowbar", "production.yml")
-        )
+        begin
+          SerializationHelper::Base.new(YamlDb::Helper).load(
+            @data.join("crowbar", "database.yml")
+          )
+        rescue SQLite3::SQLException => e
+          Rails.logger.debug "Failed to load database.yml - #{e}"
```

Comment:
Using " - " is a bit unusual... Can you write it the usual way: `Failed to load database.yml: #{e}`?